bengans_biluthyrning_test.py::imagesProducts::testText
--------------------------------------------------------------------
Last Page: file:///C:/Users/leon.loov/Documents/GitHub/Bengans-Biluthyrning/index.html
  Browser: chrome (104.0.5112.102)
   Driver: chromedriver (104.0.5112.79)
Timestamp: 1662472961  (Unix Timestamp)
     Date: Tuesday, 6 September 2022
     Time: 4:02:41 PM  (VÃ¤steuropa, sommartid, UTC+01:00)
--------------------------------------------------------------------
Traceback: File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "C:\Users\leon.loov\Documents\GitHub\Bengans-Biluthyrning\tests\bengans_biluthyrning_test.py", line 37, in testText
    self.assert_element(f"img{i}") #Checks for the 3 images
  File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\site-packages\seleniumbase\fixtures\base_case.py", line 10285, in assert_element
    self.wait_for_element_visible(selector, by=by, timeout=timeout)
  File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\site-packages\seleniumbase\fixtures\base_case.py", line 7666, in wait_for_element_visible
    return page_actions.wait_for_element_visible(
  File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\site-packages\seleniumbase\fixtures\page_actions.py", line 423, in wait_for_element_visible
    timeout_exception(NoSuchElementException, message)
  File "C:\Users\leon.loov\AppData\Local\Programs\Python\Python310\lib\site-packages\seleniumbase\fixtures\page_actions.py", line 192, in timeout_exception
    raise exc(msg)
Exception: Message: 
 Element {img#bild1} was not present after 7 seconds!
